resources/serverless/charts/webhook/values.yaml

```diff
@@ -0,0 +1,56 @@
+# Default values for webhook.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+nameOverride:
+fullnameOverride:
+
+image:
+  repository: "eu.gcr.io/kyma-project/function-webhook"
+  tag: "PR-8513"
+  pullPolicy: IfNotPresent
+
+commonLabels:
+  app: "{{ template \"webhook.fullname\" . }}"
+  version: "{{ .Values.image.tag }}"
```

Comment:
Please remove `version` and `app.kubernetes.io/version` labels due to problems with matchLabels on upgrade process.

Reply:
Good points ;) Thanks